scripts/tracker_plugin.py

```diff
@@ -36,7 +36,13 @@ class SimpleChurn(DiscoveryStrategy):
     def take_step(self):
         with self.walk_lock:
             with self.overlay.network.graph_lock:
+                to_remove = []
                 for peer in self.overlay.network.verified_peers:
+                    if time.time() - peer.last_response > 120:
+                        to_remove.append(peer)
+                for peer in to_remove:
+                    self.overlay.network.remove_peer(peer)
+                for peer in self.overlay.network.verified_peers.copy():
```

Comment:
These three lines have to be removed now:

Reply:
Ah, fixed